Automattic/wp-calypso
# Add AI feature for Woo Express plans

Related to pcUrMV-4Lc-p2#comment-16025

## Proposed Changes

* Adds `AI-assisted product descriptions` feature in plans page and in comparison grid for Woo Express Essential and Performance plans

## Testing Instructions

* Use an existing Woo Express free trial site
* Alternatively, create a Woo Express free trial site by going through NUX in https://woocommerce.com
* Go to plans page, http://calypso.localhost:3000/plans/your_site_here
* Observe `AI-assisted product descriptions` in the feature list in Essential plan grid ([screenshot](https://github.com/Automattic/wp-calypso/assets/3747241/fa778771-5ae9-4af1-9768-87b71da56534))
* Click on `Compare plans`
* Observe `AI-assisted product descriptions` in both plans compare grid under the `Products` feature group ([screenshot](https://github.com/Automattic/wp-calypso/assets/3747241/180e393f-a2c1-435c-8c9d-8f4bbab2d3c2))

## Pre-merge Checklist
